concent_api/core/queue_operations.py

```diff
@@ -0,0 +1,2 @@
+def send_blender_verification_request():  # TODO implement sending real verivication_request to Work Queue
+    pass
```

Comment:
Sending a verification request is also a part of this task. There's no issue for adding it so if you don't do it here, no one else will.

Reply:
separate issue: https://github.com/golemfactory/concent/issues/271